File: poo/structs/class.py
class Employee():

  def set_id(self, id):
    self.id = id

  def set_name(self, name):
    self.name = name

  def get_id(self):
    return self.id

  def get_name(self):
    return self.name


employee = Employee()

employee.set_id(1)
employee.set_name('Name')

print(employee.get_id())
print(employee.get_name())

File: poo/structs/class.go
package main

import "fmt"

type Employee struct {
	id   int
	name string
}

func (e *Employee) SetId(id int) {
	e.id = id
}

func (e *Employee) SetName(name string) {
	e.name = name
}

func (e *Employee) GetId() int {
	return e.id
}

func (e *Employee) GetName() string {
	return e.name
}

func main() {
	e := Employee{}
	//fmt.Printf("%v", e)
	e.id = 1
	e.name = "Name"
	//fmt.Printf("%v", e)
	e.SetId(5)
	e.SetName("Name 2")
	//fmt.Printf("%v", e)
	fmt.Println(e.GetId())
	fmt.Println(e.GetName())
}
